website/docs/src/scss/_code.scss

```diff
@@ -0,0 +1,118 @@
+/* CODE */
+
+.codeBlockContainer__-_-node_modules-\@docusaurus-theme-classic-lib-next-theme-CodeBlock-styles-module {
```

Comment:
humm maybe we can create our own component, in order to void this kind of CSS select

Reply:
We can work on this together, feel free to ping me